### PR TITLE
NAS-109134 / 12.0 / Fix the year displayed in Display System Processes and the Shell / Nas 109134

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/views/sessions/signin/signin.component.ts [hunk 1]
```diff
@@ -215,16 +215,7 @@ export class SigninComponent implements OnInit, OnDestroy, AfterViewInit {
...
-    this.http.get(`./assets/buildtime?${salt}`, {responseType: 'text'}).subscribe((res) => {
-      const buildtime = res;
-      const previous_buildtime = window.localStorage.getItem('buildtime');
-      if (buildtime !== previous_buildtime) {
-        window.localStorage.setItem('buildtime', buildtime);
-        this._copyrightYear = this.localeService.getCopyrightYearFromBuildTime();
-        window.location.reload(true);
-      }
-    });
+
```
Comment (dszidi) [inline src/app/views/sessions/signin/signin.component.ts:218]: This results in a missing copyright year

image

Comment (AlexKarpov98) [reply src/app/views/sessions/signin/signin.component.ts:218]: Need updates on middleware.
https://github.com/truenas/middleware/pull/6553

Comment (dszidi) [reply src/app/views/sessions/signin/signin.component.ts:218]: It seems those changes have been merged on middleware side

Comment (AlexKarpov98) [reply src/app/views/sessions/signin/signin.component.ts:218]: yeah, I am waiting for the building at this moment.

Comment (dszidi) [reply src/app/views/sessions/signin/signin.component.ts:218]: @AlexKarpov98 Internal 149 is ready

Comment (AlexKarpov98) [reply src/app/views/sessions/signin/signin.component.ts]: Updated

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**